controller/movies.js

```diff
@@ -0,0 +1,47 @@
+const movies = require('../movies')
+
+const getAllMovies = (request, response) => {
+  return response.send({ movies })
+}
+
+const getMoviesByTitle = (request, response) => {
+  const { title } = request.params
+  const findTitle = movies.filter((movie) => {
+    return movie.title.toLowerCase().includes(String(title))
+  })
+
+  return response.send(findTitle)
+}
+
+const getMoviesByDirector = (request, response) => {
+  const { director } = request.params
+  const findDirector = movies.filter((movie) => {
+    return movie.directors.join(' ').toLowerCase().includes(String(director))
+  })
+
+  return response.send(findDirector)
+}
```

Comment:
Combine these two functions using || to have one function that searches for title or director.